api_app/script_analyzers/classes.py

```diff
@@ -294,6 +295,19 @@ def __poll_for_result(self, req_key):
             raise AnalyzerRunException("max polls tried without getting any result.")
         return json_data
 
+    def _handle_test_run(self):
+        if hasattr(self, "is_test") and getattr(self, "is_test"):
+            # only happens in case of testing
+            self.report["success"] = True
+            return {}
```

Comment:
could this be the problem. In this case the `return` would just exit from the new function instead of the `_docker_run` function

Reply:
I got anything else about the error for Thug.

Reply:
my bad,  thank you